Source/WebCore/animation/CSSNumberishTime.cpp

```diff
@@ -105,6 +133,18 @@ CSSNumberishTime CSSNumberishTime::operator-(const CSSNumberishTime& other) cons
     return { m_type, m_value - other.m_value };
 }
 
+CSSNumberishTime CSSNumberishTime::operator*(const CSSNumberishTime& other) const
+{
+    ASSERT(m_type == other.m_type);
+    return { m_type, m_value * other.m_value };
+}
+
+CSSNumberishTime CSSNumberishTime::operator/(const CSSNumberishTime& other) const
+{
+    ASSERT(m_type == other.m_type);
+    return { m_type, m_value / other.m_value };
+}
```

Comment:
Dimensionally these don't really make sense to me. `operator*` should return like a time^2 value, and the second one a unitless value. Can / should we be using some different types here?

Reply:
I don't think we actually need `operator*` as it stands. I agree that `operator/` should return a `double` and will make that change.

Reply:
And as it turns out the only call site for `operator/` would immediately use `toDouble()` for the returned value.